criticality_score/run.py

```diff
@@ -109,7 +108,16 @@ def dependents_count(self):
                 content = result.content
                 break
             time.sleep(2**i)
-        match = DEPENDENTS_REGEX.match(content)
+        return DEPENDENTS_REGEX.match(content)
+
+    @property
+    def dependents_count(self):
+        # do query retries when no match to ensure accurate result
+        for i in range(FAIL_RETRIES):
```

Comment:
We dont need two FAIL_RETRIES and a seperate function, can you modify the loop to not break on status 200. Break only when status is 200 and there is a match as well [and return directly from inside loop].

Reply:
Yep, done